### ISSUE TITLE
Scrcpy 3.1 detected as a malware by Windows Defender 

### ISSUE THREAD

**Issue by Richard38907** [Genymobile/scrcpy#5648]
Scrcpy_win64.zip download is blocked by windows defender since v 3.1.

It works fine on 3.0.

Windows version: 11 23h2

**Comment by rom1v** [comment]
Duplicate of #5531 #5600.
